docs/content/topics/scripting/workers.md

```diff
@@ -23,7 +21,7 @@ defining project/event-specific configuration or logic.
 The remainder of this page covers the general approach of creating and using
 custom workers.
 
-# Creating a Custom Worker
+## Creating a Custom Worker
```

Comment:
Should have been an h2, so besides fixing this, everything that follows also gets bumped in one h-level.